src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/AutoFitTextureView.cs

```diff
@@ -45,17 +48,45 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
             if (ratioWidth == 0 || ratioHeight == 0)
```

Comment:
Can this ever happen? In SetAspectRatio an exception is thrown if width or height is 0

Reply:
yep can happen when the texture is measuring the size it needs to be before the camera preview is ready, ratioWidth and ratioHeight come from the camera preview dimensions

Reply:
Ah ok. Just wanted to confirm that the logic in SetAspectRatio was consistent with this